pkg/virtualKubelet/liqoNodeProvider/setup.go

```diff
@@ -80,7 +80,7 @@ func NewLiqoNodeProvider(cfg *InitConfig) *LiqoNodeProvider {
 
 func node(cfg *InitConfig) *corev1.Node {
 	lbls := map[string]string{
-		corev1.LabelHostname: cfg.NodeName,
+		corev1.LabelHostname: cfg.RemoteClusterID,
```

Comment:
Why is this change required?

Reply:
On a closer look it's not actually required - I reverted it.